cosmoz-tree.html

```diff
@@ -22,56 +29,88 @@
 			throw new Error('Must be implemented in derived object');
 		},
 
-		getPath: function (node) {
+		getPathString: function (node) {
 			throw new Error('Must be implemented in derived object');
 		},
 
-		getPathByProperty: function (propertyName, propertyValue) {
-			throw new Error('Must be implemented in derived object');
-		},
-
-		getChildren: function (node) {
+		getChildren: function (node, asList) {
 			throw new Error('Must be implemented in derived object');
 		},
 
 		getProperty: function (node, propertyName) {
 			throw new Error('Must be implemented in derived object');
 		},
 
-		getPathString: function (node, textPropertyName, separator) {
```

Comment:
This method (in PR called "getPath") should be able to live on in the "abstract class"?